oximeter/oximeter/schema/management-network-data-link.toml

```diff
@@ -0,0 +1,140 @@
+format_version = 1
+
+[target]
+name = "management_network_data_link"
+description = "A network data link related to a management gateway virtual local area network (vlan)"
```

Comment:
I'd probably say something like, "A data link on the management network". The "gateway" and "VLAN" part of this are specific to the exact timeseries we're reporting now, and there may be other link on the management network that are neither gateways nor implemented by VLANs. Those parts can be captured in the `model` and `kind` fields, I thik.